### ISSUE TITLE
a11y:  Method pane header has accessibility issues

### ISSUE THREAD

**Issue by cjfreedman** [swagger-api/swagger-ui#9576]
### Q&A (please complete the following information)
 - OS: [e.g. macOS]   Windows 10
 - Browser: [e.g. chrome, safari]  Chrome
 - Version: [e.g. 22]   121
 - Method of installation: [e.g. npm, dist assets]   npm
 - Swagger-UI version: [e.g. 3.10.0]   Swagger UI React 5.11.3
 - Swagger/OpenAPI version: [e.g. Swagger 2.0, OpenAPI 3.0]  Open API 3.0

### Content & configuration
Basic default configuration

### Describe the bug you're encountering
We use a tool called Sort Site to evaluate our site for 508 accessibility compliance.  The tool is reporting the collapsible pane for a method, e.g. a POST method, is not accessible.

### To reproduce...
The method pane for an API

### Expected behavior
No accessibility error 

### Screenshots
Here's just one of the offending elements
![image](https://github.com/swagger-api/swagger-ui/assets/86489380/b350e8a5-f1c4-4841-961c-7fe6d16c03a1)

### Additional context or thoughts
Here's the Sort Site error
![image](https://github.com/swagger-api/swagger-ui/assets/86489380/6f70dda4-20c5-46b5-9676-687717a61baf)
